sources/packages/backend/libs/sims-db/src/entities/application-disability-status.type.ts

```diff
@@ -3,6 +3,5 @@
  */
 export enum ApplicationDisabilityStatus {
```

Comment:
We can change the type of `ApplicationDisabilityStatus` to `YesNoOptions`.

Reply:
The `ApplicationDisabilityStatus` enum is shared between API and E2Es. For now, I would be okay with keeping the enum and the options as Yes/No since it would also introduce fewer changes in this PR.

Reply:
Updated